authnz/src/main/java/io/stargate/auth/PlainTextTokenSaslNegotiator.java

```diff
@@ -66,7 +66,8 @@ public boolean isComplete() {
   @Override
   public AuthenticatedUser getAuthenticatedUser() throws AuthenticationException {
     if (storedCredentials != null) {
-      return AuthenticatedUser.of(storedCredentials.getRoleName(), storedCredentials.getPassword());
+      return AuthenticatedUser.of(
+          storedCredentials.getRoleName(), storedCredentials.getPassword(), true);
```

Comment:
Do we ever use `AuthenticatedUser.of(..., ..., false)`? Would it make sense to move the reading of the `stargate.cql_use_transitional_auth` system property into this class (as a static constant, perhaps)?